Tests/YoCode_XUnit.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.2" />
     <PackageReference Include="Moq" Version="4.8.3" />
     <PackageReference Include="MSBump" Version="2.3.2" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.1.65" />
```

Comment:
Did you mean to add this? I don't think it's necessary. It's certainly not relevant to this PR. We should remove the reference to MSBump too since we're not going to use it.

Reply:
Is deleting the Nuget package is enough ?